### PR TITLE
New kickoff

### PR DESCRIPTION
**Pull request by mpicollo-wolox**
## Summary
- Add yeoman generator for new version of bootstrap

## Pending tasks
- [x] Eslint config for generator and apps should be [this](https://www.npmjs.com/package/eslint-config-wolox-node)
- [x] Missing test library option (chai or jest)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const {merge} = require('lodash'),
```
Comment (gonziesc) [inline .eslintrc.js]: this wont be needed

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
+// For a detailed explanation regarding each configuration property, visit:
```
Comment (gonziesc) [inline jest.config.js]: is this ok?

File: test/__snapshots__/optionals.spec.js.snap [hunk 3]
```diff
@@ -0,0 +1,364 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (gonziesc) [inline test/__snapshots__/optionals.spec.js.snap]: why are there snapshots here?

Comment (mpicollo-wolox) [reply test/__snapshots__/optionals.spec.js.snap]: This snapshots are used for test purposes, for check content of files generated by this kickoff, files are auto-generated by jest and updates of this works in same way (check [this link](https://jestjs.io/docs/en/snapshot-testing) for more info). Any other ideas of how test this generator are welcome.

File: jest.config.js (hunk 2 above)
Comment (mpicollo-wolox) [reply jest.config.js]: Is a config file for jest (with all configurations commented), i think that could be moved to package.json.

**Pull request closed without merging**